content/competitions/codejam-2023.html

```diff
@@ -0,0 +1,137 @@
+---
+title: "CodeJam 2023"
+date: 2022-03-14T19:13:14+10:00
```

Comment:
Nit: Date should be from this year

Reply:
See I just copy & paste the `html` file from the previous year so I have to change this manually, guess I forgot to change the date :joy: